packages/wonder-blocks-modal/components/modal-close-button.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import Color from "wonder-blocks-color";
+
+type Props = {
+    /**
+     * Whether the button icon should be dark (for light backgrounds), or light
+     * (for dark backgrounds).
+     */
+    color: "dark" | "light",
+
+    /** Called when the button is clicked. */
+    onClick?: () => void,
```

Comment:
I think I did this for parity with `onClickCloseButton`, which is also optional, to save the modal components from having to make a default behavior. But probably what should happen is that the prop should be required in both cases, and the modal components should have a no-op as a `defaultProp`, since it's _their_ weirdness that makes closing behavior not always defined at element declaration time (see other comment below).

Reply:
It's required now! The modals now have a no-op in their `defaultProps`, since the weirdness about this prop sometimes being undefined at declaration time (despite practically always being defined at execution time( is part of their API.